data/recommendation/notify.js

```diff
@@ -131,29 +130,25 @@ class Notify {
   showWhyBox() {
     this.whyBoxShowing = true;
     const lastRec = document.querySelector('.addon-box:last-child');
-    lastRec.className = 'hiddenRec';
-    lastRec.setAttribute('hidden', 'true');
+    lastRec.classList.add('hidden');
     const whyBox = document.querySelector('#why-box');
-    whyBox.removeAttribute('hidden');
-    whyBox.className = 'addon-box';
+    whyBox.classList.remove('hidden');
     const whyButton = document.querySelector('#why-button');
-    whyButton.className = 'grayed';
+    whyButton.classList.add('grayed');
   }
 
   closeWhyBox() {
     this.hideWhyBox();
-    const lastRec = document.querySelector('.hiddenRec');
-    lastRec.className = 'addon-box';
-    lastRec.removeAttribute('hidden');
+    const lastRec = document.querySelector('.addon-box:last-child');
```

Comment:
This is equivalent and fine for this PR, but eventually you'll want to not be removing a recommendation just to show the why box; there's ways to position it such that it floats above the list.